src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -112,6 +112,14 @@ bool GlRenderer::sync()
 
     mRenderPassStack.clear();
 
+    //Release disposed resources
+    if (mDisposeTextures.count > 0) {
+        for (auto i = mDisposeTextures.begin(); i < mDisposeTextures.end(); ++i) {
+            glDeleteTextures(1, i);
+        }
```

Comment:
the glDeleteTextures can delete more than one textures 
it can be like this:
```
if (mDisposeTextures.count > 0) {
    glDeleteTextures(mDisposeTextures.count, mDisposeTextures.data);
   mDisposeTextures.clear();
}
```

see https://registry.khronos.org/OpenGL-Refpages/es2.0/xhtml/glDeleteTextures.xml